File: js/wordEndings.js
// Create a function that adds a string ending to each member in an array.
// https://edabit.com/challenge/hwWLHcDoPZmXcGHyd

const addEnding = (arr, ending) => arr.map((word) => `${word}${ending}`);

module.exports = addEnding;

File: python/word_endings.py
"""
Word Endings

Create a function that adds a string ending to each member in a list.

https://edabit.com/challenge/TkL6GTu9QMhYnv869
"""
from typing import List


def add_ending(lst: List[str], ending: str) -> List[str]:
    return [f"{word}{ending}" for word in lst]
